### PR TITLE
Fix opensearch breaking schemas

### PR DESCRIPTION
**Pull request by EuroLew**
## Description

If a schema is created in opensearch according to a json file, when comparing the returned mapping from the opensearch client to the json file there should not be a schema validation error.

The issue is already fixed by opensearch on Nov 13 https://github.com/Xtansia/opensearch-java/blob/0e386f4b508b995bdfc14780ea1f2a955ad41596/java-client/src/generated/java/org/opensearch/client/opensearch/_types/mapping/JoinProperty.java, they added eager_global_ordinals to the join serialisation. The only issue is that they haven't added to a release which they appear to do every 2-3 weeks.

For now to fix the breaking issue the field will be removed from affected schemas and when an opensearch-java client version is released with the fix we can add the eager_global_ordinals option back with the updated client

## Checklist

- [x] Indices created by a schema file will not trigger an index validation error which comparing against the schema file
- [x] Add a test to catch the issue of schema files clashing against their created indices.

## Related issues

relates to #26056

### CODE REVIEW

File: zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/schema/SchemaManagerIT.java [hunk 1]
```diff
@@ -381,4 +386,23 @@ void shouldAlsoUpdateCorrespondingIndexWhenIndexTemplateUpdated(
 
     assertThat(mappingsMatch(updatedIndex.get("mappings"), newMappingsFile)).isTrue();
   }
+
+  @TestTemplate
```
Comment (ChrisKujawa) [inline zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/schema/SchemaManagerIT.java:390]: 🔧 Please mark it as regression test and link to https://github.com/camunda/camunda/issues/26056

**Review by ChrisKujawa — APPROVED** [review]
Thanks Euro great catch and thanks for the quick fix 🚀 

Can you add to the PR description a bit more details (the information you shared to me, regarding OS bug fix and release, etc. that we should bring it back) Furthermore, create an follow up issue for bringing this back, please.

**Comment by ChrisKujawa** [comment]
@EuroLew somehow our test is failing 🤔

**Comment by EuroLew** [comment]
> @EuroLew somehow our test is failing 🤔

fixed @RegressionTest is not compatible with @TestTemplate by adding @RegressionTestTemplate

**Pull request merged**